astroid/brain/brain_typing.py

```diff
@@ -161,7 +179,7 @@ def _looks_like_typing_alias(node: nodes.Call) -> bool:
         isinstance(node, nodes.Call)
         and isinstance(node.func, nodes.Name)
         and node.func.name == "_alias"
-        and isinstance(node.args[0], nodes.Attribute)
+        and isinstance(node.args[0], (nodes.Attribute, nodes.Name))
```

Comment:
This change is necessary, but causes many tests to fail (pylint acceptance "typing", astroid). Might be related to the brain for `builtins`.

Reply:
@cdce8p you are right it is necessary. I found a fix for this. It will be part of my next push on #921.

Reply:
@hippo91 Sounds good! I'm looking forward to it.